docs/source/release_notes.rst

```diff
@@ -8,6 +8,8 @@ Future Release
     * Enhancements
     * Fixes
         * Fix dependency issues (:pr:`2644`, :pr:`2656`)
+        * Add workaround for pandas 2.2.0 bug with nunique and unpin pandas (:pr:`2657`)
+        * Fix the tests that run with Woodwork main so they can be triggered (:pr:`2657`)
```

Comment:
Nit: I think I'd put this in the Testing Changes section

Reply:
done!